### ISSUE TITLE
When ordering decimal quantities, item is removed from cart without warning.

### ISSUE THREAD

**Issue by sstead** [openfoodfoundation/openfoodnetwork#659]
A customer Tasmania raised this issue. She was trying to order .5kg of a product listed as 1kg. The price on the right calculated properly, and half an item is added to the cart. But when she clicks edit cart, the item is removed without any warning.

Options:
- Either we need to create an alert that tells them 'this item is only sold in 1kg units' 
- Or we can allow decimal ordering 

Tim has raised a similar issue for his buying group... http://community.openfoodnetwork.org/t/bulk-and-group-buying-wishlist/101/15

**Comment by lin-d-hop** [comment]
I explored decimal ordering when exploring variable weights. Discrete order units is a deep assumption in Spree and when exploring it quickly became clear that we would be dealing with a web of rounding and pricing issues if we started to mess around making order units continuous. 

Might be interesting to look into again after an upgrade of spree, but in the mean time I think some kind of alert would be a better option on this one.
